lib/rubocop/ast/node_pattern/compiler/binding.rb

```diff
@@ -26,8 +26,9 @@ def bind(name)
             var
           end
 
-          # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-          def union_bind(enum)
+          # Yields for each branch of the given union, forbidding unification of
+          # bindings which only appear in a subset of the union.
+          def union_bind(enum) # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
I'm not familiar with this section of the code base and what this method is actually meant to do, so I'm not sure this comment is accurate.